### ISSUE TITLE
wmic cpu get addresswidth,caption,manufacturer /FORMAT:csv

### ISSUE THREAD

**Issue by AnonimPython** [reflex-dev/reflex#4198]
**Describe the bug**
The command `wmic cpu get addresswidth,caption,manufacturer /FORMAT:csv` failed with error: Command 'wmic cpu get
addresswidth,caption,manufacturer /FORMAT:csv' returned non-zero exit status 1.. This will return None.

AND infinity reloading local server!

**Specifics (please complete the following information):**
 - Python Version: 3.12
 - Reflex Version: 0.6.3
 - OS: Windows 11
 - Browser (Optional): Chrome
 
**Additional context**
After create a new project and first start server it's work, but after why I remake the code server are not starting , just say in terminal "WARNING:  WatchFiles detected changes in 'FistApp\main.py'. Reloading..."
after this my terminal are crashing, not working. I need to close him and open then.

**Comment by masenf** [comment]
is this a regression from 0.6.2?

**Comment by AnonimPython** [comment]
> [ENG-3962 wmic cpu get addresswidth,caption,manufacturer /FORMAT:csv](https://linear.app/reflex-dev/issue/ENG-3962/wmic-cpu-get-addresswidthcaptionmanufacturer-formatcsv)

I for Russia. Site doesn't work( 
Do u have alterative?

**Comment by AnonimPython** [comment]
> is this a regression from 0.6.2?

idk, but I can say. 6 month ago I dont take this problem.

**Comment by masenf** [comment]
> Do u have alternative?

That's an internal issue tracker for the team, it's not public. Any comments here will end up over there and we'll triage it for an upcoming sprint.

**Comment by adhami3310** [comment]
Can you try running:

```bash
wmic cpu get addresswidth,caption,manufacturer /FORMAT:csv
```

in the command line?

otherwise, the infinite hot-reload isn't because of this, but because of something else (unsure as we don't have any clue towards that), can you try running with:

```bash
reflex run --loglevel prod
```

**Comment by mh-maker** [comment]
Not sure if this helps (I'm new to reflex and web development, in general. But the minimal config demo on a windows notebook also creates the error message:

`Ungültiges XSL-Format (oder) Dateiname.
The command `wmic cpu get addresswidth,caption,manufacturer /FORMAT:csv` failed with error: Command 'wmic cpu get addresswidth,caption,manufacturer /FORMAT:csv' returned non-zero exit 
status 44210.. This will return None.
`
I.e. I didn't change anything in this repo. Does this help?

**Comment by masenf** [comment]
when you just run `wmic cpu get addresswidth,caption,manufacturer /FORMAT:csv` in the terminal, is there an error message that comes up?

**Comment by mh-maker** [comment]
Yes, it says:

Ungültiges XSL-Format (oder) Dateiname.

Which essentially translates into "Invalid XLS-format or filename"

**Comment by masenf** [comment]
interesting, seeing this http://www.thegeekblog.co.uk/quick-tips/2013/10/8/fix-the-invalid-xsl-format-or-file-name-error.html

which suggests this may be a problem with the `/format` switch in conjunction with a locale other than the one used when installing windows...

does this error prevent you from running the app? or does it just appear when you start up?

**Comment by mh-maker** [comment]
No, it does not prevent me from running the app, it just appears when I start it up.

**Comment by mh-maker** [comment]
I just did what is mentioned in the post you shared above and it worked! Thanks a lot! And on my pc this is exactly the case that I'm using a different locale then the one that was originally installed.

**Comment by masenf** [comment]
Cool, glad that workaround worked for you.

Long term reflex needs to get away from the `wmic.exe` command line tool because it was deprecated in windows 11 and not installed by default in some cases, so we can't rely on it.
